### PR TITLE
sig network kops job for beta features

### PR DESCRIPTION
**Pull request by aojea**
It needs to be generated, but it turns out the makefile requires access to aws that I do not have

/assign @justinsb

### CODE REVIEW

File: config/jobs/kubernetes/kops/build_jobs.py [hunk 1]
```diff
@@ -925,6 +925,23 @@ def generate_misc():
                    runs_per_day=3),
 
 
+        # [sig-network, @aojea] A job to test SIG Network beta features on Kops to validate "real-world" usage availability.
```
Comment (aojea) [inline config/jobs/kubernetes/kops/build_jobs.py:928]: I will like if I can receive also alerts from this job, so I can be notified on failures

Comment (justinsb) [reply config/jobs/kubernetes/kops/build_jobs.py:928]: Not sure how to do that TBH - does prow support this?

**Comment by justinsb** [comment]
/approve
/lgtm

Looks good, these are a bit tricky but we can iterate if anything is wrong

File: config/jobs/kubernetes/kops/build_jobs.py (hunk 1 above)
Comment (aojea) [reply config/jobs/kubernetes/kops/build_jobs.py:928]: https://github.com/kubernetes/test-infra/blob/7ba5dbe703fbd4bdf3c693a7e24972634703c215/config/jobs/kubernetes-sigs/kind/kind.yaml#L77

**Pull request merged**